runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -756,10 +756,17 @@ J9::ARM64::TreeEvaluator::multianewArrayEvaluator(TR::Node *node, TR::CodeGenera
 TR::Register *
 J9::ARM64::TreeEvaluator::newObjectEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    {
-   TR::ILOpCodes opCode = node->getOpCodeValue();
-   TR::Node::recreate(node, TR::acall);
-   TR::Register *targetRegister = directCallEvaluator(node, cg);
-   TR::Node::recreate(node, opCode);
+   TR::Compilation *comp = cg->comp();
```

Comment:
`comp` is used nowhere in this function.

Reply:
Removed unused `comp`.